### PR TITLE
Add tests wk 1

### PR DESCRIPTION
**Pull request by EmilePW**
Added some tests in the same vein as from the JS Core 1 homework.These give a bit more feedback about what might have gone wrong if the code doesn't work and tests edge cases. I'm hoping it will make the homework quicker to mark as well.

### CODE REVIEW

**Comment by EmilePW** [comment]
@ChrisOwen101 - sorry, forgot to actually add a notification for this yesterday.

File: Week-1/Homework/mandatory/1-writers.js [hunk 1]
```diff
@@ -1,3 +1,14 @@
+/* ======= TESTS - DO NOT MODIFY =====
```
Comment (ChrisOwen101) [inline Week-1/Homework/mandatory/1-writers.js]: Could this be extracted to a separate JS file so we don't have to include it at the top of the file where it's quite invasive?

File: Week-1/Homework/mandatory/3-groceries.js [hunk 2]
```diff
@@ -1,3 +1,15 @@
+/* ======= TESTS - DO NOT MODIFY =====
```
Comment (ChrisOwen101) [inline Week-1/Homework/mandatory/3-groceries.js]: Same as the above - could this be moved into a JS file that can be included here? Also reduced duplicated code

File: Week-1/Homework/mandatory/1-writers.js [hunk 3]
```diff
@@ -36,9 +47,59 @@ let writers = [
...
+Make sure to write the code for this underneath the code for the first task
 */
+
+/* ======= TESTS - DO NOT MODIFY ===== 
+There are some Tests in this file that will help you work out if your code is working.
+
+To run these tests type `node 1-writers.js` into your terminal or run in VS code (F5)
+*/
+
+function test(test_name, expr, expected, actual) {
```
Comment (ChrisOwen101) [inline Week-1/Homework/mandatory/1-writers.js]: Maybe this function could be pulled out to another JS file as well?

**Review by ChrisOwen101 — APPROVED** [review]
Great work! A few small comments that would help tidy this up

**Comment by EmilePW** [comment]
Thanks for reviewing @ChrisOwen101. I was initially thinking it might cause problems if I start using `require` depending on how they are running the code but I think it's fine as long as they've followed the CYF instructions on setting things up. I'll make the changes you suggested above later.

**Comment by EmilePW** [comment]
Additional changes made above @ChrisOwen101
